docs/podman-varlink.1.md

```diff
@@ -6,34 +6,43 @@
 podman\-varlink - Runs the varlink backend interface
 
 ## SYNOPSIS
-**podman varlink**
+**podman varlink** [OPTIONS]
 [**--help**|**-h**]
 VARLINK_URI
 
 ## DESCRIPTION
-Starts the varlink service that allows varlink clients to interact with podman.
-<!--
-More will go here as the docs and api firm up.
--->
+Starts the varlink service that allows varlink clients to interact with podman.  This should generally be done
+with systemd.  See _Configuration_ below.
 
-**podman [GLOBAL OPTIONS] varlink **
+**podman [GLOBAL OPTIONS] [OPTIONS] varlink VARLINK_URI**
```

Comment:
Shouldn't this be:

    podman [GLOBAL OPTIONS] varlink [OPTIONS] VARLINK_URI

And shouldn't it go in the SYNOPSIS section above?